engine/apps/base/models/live_setting.py

```diff
@@ -174,8 +188,21 @@ def populate_settings_if_needed(cls):
         settings_in_db = cls.objects.filter(name__in=cls.AVAILABLE_NAMES).values_list("name", flat=True)
         setting_names_to_populate = set(cls.AVAILABLE_NAMES) - set(settings_in_db)
 
+        revalidate_twilio = False
         for setting_name in setting_names_to_populate:
-            cls.objects.create(name=setting_name, value=cls._get_setting_from_setting_file(setting_name))
+            setting = cls.objects.create(name=setting_name, value=cls._get_setting_from_setting_file(setting_name))
+            if setting.name.startswith("TWILIO"):
+                revalidate_twilio = True
+
+        if revalidate_twilio:
+            cls.revalidate_twilio()
```

Comment:
Certain Twilio env vars depend on each other for validation, when resetting one value the error would remain on a another unchanged value that is now OK.

Would it make sense to re-validate all everytime rather than having this specific to Twilio parameters?